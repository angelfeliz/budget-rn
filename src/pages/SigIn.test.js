import React from 'react';
import renderer from 'react-test-renderer';
import SigIn from './SignIn';

it('render correctly', () => {
    const tree = renderer.create(<SigIn/>);

    expect(tree).toMatchSnapshot();
})