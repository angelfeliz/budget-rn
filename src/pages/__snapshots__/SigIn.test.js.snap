// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#000",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
        "paddingHorizontal": 10,
      }
    }
  >
    <Text>
      Sign In
    </Text>
    <View
      style={
        Object {
          "flexDirection": "column",
          "justifyContent": "flex-start",
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "column",
            "height": 130,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFF",
            }
          }
        >
          Username
        </Text>
        <TextInput
          allowFontScaling={true}
          onChangeText={[Function]}
          rejectResponderTermination={true}
          style={
            Object {
              "borderBottomColor": "#FFF",
              "borderWidth": 1,
              "color": "#FFF",
              "height": 10,
              "marginBottom": 0,
              "paddingTop": 20,
            }
          }
          underlineColorAndroid="transparent"
          value=""
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
            "height": 130,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFF",
            }
          }
        >
          Password
        </Text>
        <TextInput
          allowFontScaling={true}
          onChangeText={[Function]}
          rejectResponderTermination={true}
          style={
            Object {
              "borderBottomColor": "#FFF",
              "borderWidth": 1,
              "color": "#FFF",
              "height": 10,
              "marginBottom": 0,
              "paddingTop": 20,
            }
          }
          underlineColorAndroid="transparent"
          value=""
        />
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "flex-end",
          "marginBottom": 36,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#DDDDDD",
            "height": 50,
            "justifyContent": "center",
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text>
          Sign In
        </Text>
      </View>
    </View>
  </View>
</View>
`;
